extensions/positron-r/src/testing/testing.ts

```diff
@@ -13,7 +13,16 @@ import { detectRPackage, getRPackageName } from '../contexts';
 
 let controller: vscode.TestController | undefined;
 
+const _onDidDiscoverTestFiles = new vscode.EventEmitter<void>();
+
+/**
+ * An event that fires after the test files are discovered.
+ */
+export const onDidDiscoverTestFiles = _onDidDiscoverTestFiles.event;
+
+
 export async function setupTestExplorer(context: vscode.ExtensionContext) {
+	context.workspaceState.update('testExplorerSetUp', false);
```

Comment:
If this is a global workspace state, should we somehow namespace this to positron-r? In Positron Assistant, I see a key like this: `'positron-assistant.recentFiles'`.